roles/maas_postgres/tasks/install_postgres.yaml

```diff
@@ -82,11 +82,23 @@
   ansible.builtin.template:
     src: pgsql_check.j2
     dest: /opt/pgsql_check
-    mode: 0711
+    mode: 0755
```

Comment:
this file has the postgres password in plain-text, does it needs to be world-readable?

Reply:
It does, xinetd runs as `nobody` and requires read access to the script before executing it.

Reply:
:man_shrugging: